manifests/0000_70_cluster-network-operator_04_daemonset.yaml

```diff
@@ -0,0 +1,157 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: network-plugin-install
+  namespace: openshift-network-operator
+  annotations:
+    kubernetes.io/description: |
+      This daemonset installs all necessary CNI binaries on every host across the cluster
+spec:
+  selector:
+    matchLabels:
+      app: network-plugin-install
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: network-plugin-install
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      priorityClassName: "system-node-critical"
+      initContainers:
+      - name: multus-bin-copy
+        image: "quay.io/openshift/origin-multus-cni:4.3"
```

Comment:
This needs proper release image substitution, like all other image references.

Reply:
This is proper release image substitution for assets in `manifests`. We can't use template variables here. But CI understands this completely fine.

Reply:
for CVO-deployed objects, but this shouldn't be CVO-managed.